php-checks/src/main/java/org/sonar/php/checks/formatting/IndentationCheck.java

```diff
@@ -101,9 +101,9 @@ private void checkClosingParenthesisLocation(FormattingStandardCheck formattingC
       rParenthesis = paramList.getParent().getFirstChild(PHPPunctuator.RPARENTHESIS);
       msg = "Move the closing parenthesis with the opening brace on the next line.";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "msg". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1854)